packages/components/src/components/tag/tag.stories.ts

```diff
@@ -29,173 +34,86 @@ export const Default = {
 };
 
 /**
- * The tag in all possible combinations of `selected` and `size`.
+ * Use the `size` attribute to change the tag size. The default size is `lg`.
  */
 
-export const selectedAndSize = {
-  name: 'Selected × Size',
-  parameters: { controls: { exclude: ['size', 'selected'] } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        x: { type: 'attribute', name: 'selected', values: ['false', 'true'] },
-        y: { type: 'attribute', name: 'size' }
-      },
-      args
-    });
-  }
+export const Size = {
+  render: () => html`
+    <div class="flex gap-12">
+      <sd-tag size="lg">Large</sd-tag>
+      <sd-tag size="sm">Small</sd-tag>
+    </div>
+  `
 };
 
 /**
- * The tag in all possible combinations of `removable` and `size`.
+ * Use the `selected` attribute to enable the selected state.
  */
 
-export const removableAndSize = {
-  name: 'Removable × Size',
-  parameters: { controls: { exclude: ['size', 'removable'] } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        x: { type: 'attribute', name: 'removable', values: ['false', 'true'] },
-        y: { type: 'attribute', name: 'size' }
-      },
-      args
-    });
-  }
+export const Selected = {
+  render: () => html`
+    <div class="flex gap-12">
+      <sd-tag selected>Selected</sd-tag>
```

Comment:
In Figma File the label ist "Large".

Reply:
<img width="628" alt="image" src="https://github.com/user-attachments/assets/32edbb44-4593-4484-bfea-d297a66c30bd">

Reply:
Fixed in Figma. Label is now "Selected"

Reply:
Resolved.